pychonet/lib/const.py

```diff
@@ -67,9 +67,16 @@
     0x5E: {'name': 'SetGet_SNA', 'description': 'Property value write & read (response not possible)'}
 }
 
+MANUFACTURERS = {
```

Comment:
I can see an argument to not map this to a string but instead return the Id. Thoughts?